lib/oga/xml/element.rb

```diff
@@ -64,6 +64,8 @@ def namespace_name=(name)
       #
       # @return [Oga::XML::Attribute]
       def attribute(name)
+        name = name.to_s if name.is_a?(Symbol)
+
```

Comment:
These changes do not belong in this pull request. The method `split_name` already takes care of casting things to a String.